README.md

```diff
@@ -1,11 +1,45 @@
 # rectangle-overlap
 
+[![TypeScript support](https://img.shields.io/npm/types/rectangle-overlap.svg)](https://github.com/lovasoa/rectangle-overlap/blob/master/index.ts)
+[![License](https://img.shields.io/npm/l/rectangle-overlap.svg)](https://github.com/lovasoa/rectangle-overlap/blob/master/LICENSE)
+
 Fastly compute the area of the intersection of two rectangles.
 
 ## Usage
 
 ```js
-var area = require("rectangle-overlap");
+const area = require("rectangle-overlap");
```

Comment:
Now that the function returns a rectangle and not a number, it might be more judicious to call it `intersection` than `area`.